src/material-examples/ripple-overview/ripple-overview-example.css

```diff
@@ -0,0 +1,20 @@
+.example-ripple-container {
+  /** Necessary until: https://github.com/angular/material2/pull/11913*/
+  position: relative;
+
+  cursor: pointer;
+  text-align: center;
+
+  width: 300px;
+  height: 300px;
+  line-height: 300px;
+}
+
+/** Styles to make the demo look better. */
+mat-checkbox {
+  margin: 6px 12px 6px 0;
+}
+
+mat-form-field {
```

Comment:
Can you use css classes for these styles instead of styling the elements directly?

Reply:
Done. I didn't intend to use a class here because I felt like it would bloat the example HTML and since we have view encapsulation turned on for the examples, it wouldn't affect other things.

Reply:
This is something I insist on because it makes it easier for people inspecting the demos to know the difference between the component styles and the example styles.